### DISCUSSION TITLE
Why does prism launcher doesn't work skin in Offline account?

### DISCUSSION THREAD

**Discussion by MinecraftNguyen** [PrismLauncher/PrismLauncher#2261] [Ideas]
Now, I don't have money and I play offline account and add skin. But, it can't work at the offline account.In future, can you add skin for Offline account?

![Skin_not_work_at_offline_account](https://github.com/PrismLauncher/PrismLauncher/assets/164752324/7e051dec-cb1a-46af-ac94-35bada60b260)

**Comment by Trial97** [comment]
Because it only uses the official way to change the skins and that only works for online accounts.
